frontend/src/components/Field/YesNo/YesNo.tsx

```diff
@@ -0,0 +1,169 @@
+import { KeyboardEvent, useCallback, useMemo } from 'react'
+import { BiCheck, BiX } from 'react-icons/bi'
+import {
+  Box,
+  forwardRef,
+  HStack,
+  Icon,
+  useRadio,
+  useRadioGroup,
+  UseRadioGroupProps,
+  UseRadioGroupReturn,
+  UseRadioProps,
+  useStyleConfig,
+  VisuallyHidden,
+} from '@chakra-ui/react'
+
+import { YESNO_THEME_KEY } from '~theme/components/Field/YesNo'
+import { FieldColorScheme } from '~theme/foundations/colours'
+
+export interface YesNoProps {
```

Comment:
Is there a reason why we don't extend `UseRadioGroupProps`? Is it to make explicit what is passed into this component?

Reply:
Yes, we don't actually need those props. We want to be slightly stricter with this component